common/test/com/commercetools/sunrise/framework/i18n/I18nResolverTest.java

```diff
@@ -0,0 +1,95 @@
+package com.commercetools.sunrise.framework.i18n;
+
+import io.sphere.sdk.models.LocalizedString;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonMap;
+import static java.util.Locale.ENGLISH;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class I18nResolverTest {
+
+    private static final String MESSAGE_KEY = "foo.bar";
+    private static final String MESSAGE_ANSWER = "OK";
+    private static final Map<String, Object> ARGS = singletonMap("arg1", "val1");
+
+    @Mock(answer = Answers.CALLS_REAL_METHODS)
```

Comment:
This partial mocking looks very useful 👍

Reply:
Definitely! I was checking because partial mocking is considered a bad practice in many cases but definitely for testing default methods in interfaces is very useful.